<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlexditorMainWindow</class>
 <widget class="QMainWindow" name="AlexditorMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AlexditorMainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="ouvrir_pushButton">
        <property name="text">
         <string>Ouvrir</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nouveau_pushButton">
        <property name="text">
         <string>Nouveau</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="sauver_pushButton">
        <property name="text">
         <string>Sauver</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sauverSous_pushButton">
        <property name="text">
         <string>Sauver sous...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>textEdit</tabstop>
  <tabstop>ouvrir_pushButton</tabstop>
  <tabstop>nouveau_pushButton</tabstop>
  <tabstop>sauver_pushButton</tabstop>
  <tabstop>sauverSous_pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
